sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -954,15 +953,7 @@ def get_response_encoding():
 
 
 def _verify_interpreter_version_is_supported(pipeline_options):
-  if sys.version_info[0] == 2:
-    return
-
-  if sys.version_info[0:2] in [(3, 5), (3, 6)]:
-    if sys.version_info[0:3] < (3, 5, 3):
-      warnings.warn(
-          'You are using an early release for Python 3.5. It is recommended '
-          'to use Python 3.5.3 or higher with Dataflow '
-          'runner.')
+  if sys.version_info[0:2] in [(2, 7), (3, 5), (3, 6), (3, 7)]:
     return
 
   debug_options = pipeline_options.view_as(DebugOptions)
```

Comment:
Do we still need `ignore_py3_minor_version` experiment?

Reply:
It may be helpful to force the SDK to start a Dataflow job on, say, python 3.8 when it becomes available. I don't have a strong opinion whether to keep it or not.

Reply:
Let's keep it for now, in case you want to get this PR into the release branch.

Python 3.8, probably we will need to add explicit support by changing this list anyway.